docs/guide/seo.rst

```diff
@@ -0,0 +1,57 @@
+================================
+Search Engine Optimization (SEO)
```

Comment:
```suggestion
Search Engine Optimization (SEO) for documentation
```

Reply:
optional, but considering the SEO of this page itself, it could do to have a more descriptive title

Reply:
:+1: